nature/templates/nature/reports/species-regulations-report.html

```diff
@@ -25,20 +25,24 @@
         <div class="col-sm-10">
             {{ species.name_fi | lower | capfirst }}
             <br/>
-            <i>{{ species.name_sci_1 | lower | capfirst }}</i>
+            {% if species.name_sci_1 %}
+                <i>{{ species.name_sci_1 | lower | capfirst }}</i>
+            {% endif %}
             <br/>
-            <i>{{ species.name_subspecies_1 | lower | capfirst }}</i>
+            {% if species.name_subspecies_1 %}
+                <i>{{ species.name_subspecies_1 | lower | capfirst }}</i>
+            {% endif %}
         </div>
```

Comment:
There will be extra `<br>`s here when these are false. Does it look weird in the UI?